templates/services.yaml

```diff
@@ -80,6 +80,24 @@ metadata:
   name: {{ .Release.Name }}-st2web{{ template "enterpriseSuffix" . }}
   annotations:
     description: StackStorm st2web, - an admin Web UI and main entry point for external API requests
+    {{- if .Values.st2web.service.hostname }}
+    external-dns.alpha.kubernetes.io/hostname: {{ .Values.st2web.service.hostname | quote }}
+    {{- end }}
+    # For more information, see https://kubernetes.io/docs/concepts/services-networking/service/#ssl-support-on-aws
+    {{- if eq .Values.st2web.service.type "LoadBalancer" }}
+    {{- if .Values.st2web.service.loadBalancer.sslCert }}
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: {{ .Values.st2web.service.loadBalancer.sslCert | quote }}
```

Comment:
That's not exactly what user will be looking. The problem is that there could be _much_ more possible settings to configure via annotations which we can't template and cover.

Just expose `annotations` in values.yaml and let them decide what's needed.
This is a common practice, according to Helm charts repo.

Something like https://github.com/helm/charts/blob/master/stable/cockroachdb/values.yaml#L47-L49